types/filter.d.ts

```diff
@@ -14,5 +14,13 @@ export declare function filterToParams(
     nameAlias: Record<string, string>,
     nameFunc: Record<string, () => unknown>,
     filterFields: Record<string, string>,
-    keywordFields: Record<string, string[]>
+    keywordFields: Record<string, string[]>,
+    {
+        imperfectFilterFields,
+        keywords
+    }: {
+        imperfectFilterFields: Record<string, string>;
+        keywords: Record<string, () => unknown>;

```

Comment:
will the return always be unknown or can we assume it can be of a certain type(s)?

Reply:
unknown